File: mullins-roger/change-back.js
// cents / .25 = quarters
// cents = cents - (quarters)
// cents / .10 = dimes
// cents = cents - (dimes)
// cents / .05 = nickels
// cents = cents - (nickels)
// cents = pennies

function coinCombo (cents) {
  let totalIn = cents;
  let quarters = Math.floor(cents/25);
  totalIn = totalIn - (quarters * 25);
  console.log("Total in = " + cents); 
  console.log("Number of quarters: " + quarters);
  console.log("New total:" + totalIn);
  
  let dimes = Math.floor(totalIn/10);
  totalIn = totalIn - (dimes * 10);
  console.log("Total in = " + cents); 
  console.log("Number of dimes: " + dimes);
  console.log("New total:" + totalIn);
  
  let nickels = Math.floor(totalIn/5);
  totalIn = totalIn - (nickels * 5);
  console.log("Total in = " + cents); 
  console.log("Number of nickels: " + nickels);
  console.log("New total:" + totalIn);
  
  let pennies = totalIn;
  totalIn = totalIn - (pennies);
  console.log("Total in = " + cents); 
  console.log("Number of pennies: " + pennies);
  console.log("New total:" + totalIn);
  
  return [pennies, nickels, dimes, quarters];
}

File: mullins-roger/change-back.py
def coin_combo(cents):
    coins = [0,0,0,0]
    print("In: ", cents)
    coins[3] = cents // 25
    print(cents, "// 25 = ", coins[3])
    cents = cents - coins[3]*25
    print("Cents is now: ", cents)
    coins[2] = cents // 10
    print(cents, "// 10 = ", coins[2])
    cents = cents - coins[2]*10
    print("Cents is now: ", cents)
    coins[1] = cents // 5
    print(cents, "// 5 = ", coins[2])
    cents = cents - coins[1]*5
    print("Cents remaining: ", cents)
    coins[0] = cents
    return coins
